generators/csharp/codegen/src/asIs/RawClient.Template.cs

```diff
@@ -110,23 +86,68 @@ public record ApiResponse
 
         public required HttpResponseMessage Raw { get; init; }
     }
+ 
+    private async Task<ApiResponse> SendWithRetriesAsync(
+        BaseApiRequest request,
+        CancellationToken cancellationToken
+    )
+    {
+        var httpClient = request.Options?.HttpClient ?? Options.HttpClient;
+        var maxRetries = request.Options?.MaxRetries ?? Options.MaxRetries;
+        var response = await httpClient.SendAsync(BuildHttpRequest(request), cancellationToken);
+        for (var i = 0; i < maxRetries; i++)
+        {
+            if (!ShouldRetry(response))
+            {
+                break;
+            }
+            var delayMs = Math.Min(InitialRetryDelayMs * (int)Math.Pow(2, i), MaxRetryDelayMs);
+            await System.Threading.Tasks.Task.Delay(delayMs, cancellationToken);
```

Comment:
Note that this is the fully-qualified function name for `Task.Delay` to handle the case when the user defines a `Task` type (i.e. the `csharp-namespace-conflict` test definition). 

If there are more conflicts later on, we can refactor the `RawClient` so it's not an as-is file, but the benefits of keeping this file as-is outweighs the potential conflict with other types for now (especially because they're very unlikely).